libcxx/utils/libcxx/test/dsl.py

```diff
@@ -69,8 +69,14 @@ def f(config, *args, **kwargs):
             if cacheKey not in cache:
                 cache[cacheKey] = function(config, *args, **kwargs)
                 # Update the persistent cache so it knows about the new key
-                with open(persistentCache, "wb") as cacheFile:
+                # We write to a PID-suffixed file and rename the result to
```

Comment:
That definitely works for me, but the comment at the top of the file suggests that reusing it between runs is important:

```
 We also store the cache for a given function persistently across invocations
    of Lit. This dramatically speeds up the configuration of the test suite when
    invoking Lit repeatedly, which is important for developer workflow.
```

Adding the PID would break this re-use so I'm not sure if that is acceptable?

Reply:
Oh, right -- then we'd basically lose this optimization entirely. OK, I guess I am fine with the current approach then.